getting-started/pull-request-lifecycle.rst

```diff
@@ -542,6 +542,18 @@ If you identify such flaky behavior, look for an issue in the `issue tracker`_
 that describes this particular flakiness. Create a new issue if you can't
 find one.
 
+:guilabel:`Update branch` button
+================================
+
+You can click on the :guilabel:`Update branch` button to merge the latest
```

Comment:
We might want to add something like: If the PR is very old, it may be useful to update the branch before merging to ensure that the PR does not fail any CI checks that were added or changed since CI last ran.

We occasionally get issues if we have old PRs that fail a newly added check.